### PR TITLE
DOC, INFRA, CI: document and bump versions

### PR DESCRIPTION
**Pull request by sappelhoff**
follow up to #267

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -69,9 +69,7 @@ All required dependencies are installed automatically when you install ``autorej
 
 Optional dependencies are:
 
-* ``tqdm`` (for nice progress-bars when setting ``verbose=True``)
-* ``h5io`` (for writing ``autoreject`` objects using the HDF5 format)
```
Comment (sappelhoff) [inline README.rst:73]: tqdm and h5io are now part of mne deps, if we require mne >= 1.0 and mne[hdf5], ... that's why we can remove this here IMHO

File: setup.py [hunk 2]
```diff
@@ -52,25 +52,20 @@
...
-                  'tqdm',
-                  'h5io'
-              ],
               'test': [
                   'pytest',
                   'pytest-cov',
                   'pytest-sugar',
                   'check-manifest',
                   'flake8',
-                  'pooch'
```
Comment (sappelhoff) [inline setup.py:73]: pooch is now shipped with mne

**Review by agramfort — APPROVED** [review]
thx for the spring cleanup !

**Comment by jasmainak** [comment]
Thanks @sappelhoff !

Just one comment, the PR was from upstream rather than the fork ;-)

**Pull request merged**